### PR TITLE
Removed 'Canary' from "Remote debugging wv2 WinUI 2 (UWP) apps"

### LINKED ISSUE #2532: Is the Canary Channel of Edge necessary for remote debugging of WebView2 on UWP?
This documentation recommends that you download the canary channel of Microsoft Edge in step 1. Is this really necessary to be able to follow the rest of the steps? It seems to work fine with the version I have-- 111.0.1661.62 (Official build) (64-bit)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c27cd959-ebbb-089e-2b4c-ac1548a09093
* Version Independent ID: c27cd959-ebbb-089e-2b4c-ac1548a09093
* Content: [Remote debugging WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge - Microsoft Edge Development](https://learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/remote-debugging)
* Content Source: [microsoft-edge/webview2/how-to/remote-debugging.md](https://github.com/MicrosoftDocs/edge-developer/blob/main/microsoft-edge/webview2/how-to/remote-debugging.md)
* Product: **microsoft-edge**
* Technology: **webview**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

### PR DESCRIPTION
**Pull request by mikehoffms**
This PR fixes #2532.
* Removes mention of Canary.
* Removes setting the channel preference.

Rendered article:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/remote-debugging?branch=pr-en-us-2546

gh rendered:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/remote-debug/microsoft-edge/webview2/how-to/remote-debugging.md

### CODE REVIEW

**Review by maurawinstanley — APPROVED** [review]

File: microsoft-edge/webview2/how-to/remote-debugging.md [hunk 1]
```diff
@@ -20,9 +20,7 @@ Remote debugging is necessary for this platform because currently, the built-in
 
 Attach Microsoft Edge DevTools remotely to a WebView2 WinUI 2 (UWP) app as follows:
 
-1.  Install [Microsoft Edge Canary Channel](https://www.microsoftedgeinsider.com/download/canary).
-
-1.  In Microsoft Edge Canary, go to to `edge://inspect` and verify that the **Inspect with Edge Developer Tools** page exists in your version of the browser:
+1.  In Microsoft Edge, go to `edge://inspect`.  The **Inspect with Edge Developer Tools** page opens:
 
     ![DevTools Inspect utility page](./remote-debugging-images/inspect-devtools-page-supported.png)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/remote-debugging.md:26]: @maurawinstanley - is this still ok?  WEBVIEW2_RELEASE_CHANNEL_PREFERENCE=1

Step 11 in gh rendered section:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/remote-debug/microsoft-edge/webview2/how-to/remote-debugging.md#debugging-desktop-webview2-winui-2-uwp-apps

Comment (maurawinstanley) [reply microsoft-edge/webview2/how-to/remote-debugging.md:26]: We can remove anything related to setting the release channel preference to 1, this is no longer needed.

Comment (maurawinstanley) [reply microsoft-edge/webview2/how-to/remote-debugging.md:26]: We can remove anything related to setting the release channel preference to 1, this is no longer needed.

Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/remote-debugging.md:26]: my commit that removes setting release channel preference:
https://github.com/MicrosoftDocs/edge-developer/pull/2546/commits/a9953711a02a7ed96176e57624f27fc02f17ab02

**Pull request merged**